translation/translator.go

```diff
@@ -81,37 +82,38 @@ func (t *Translator) Get(key string, options ...translationcontract.Option) stri
 		fallback = *options[0].Fallback
 	}
 
-	// Parse the key into group and key parts.
-	group, keyPart := parseKey(key)
-
 	// For JSON translations, there is only one file per locale, so we will
 	// simply load the file and return the line if it exists.
 	// If the file doesn't exist, we will return fallback if it is enabled.
 	// Otherwise, we will return the key as the line.
-	if err := t.load(locale, group); err != nil && err != ErrFileNotExist {
+	if err := t.load(locale, "*"); err != nil && err != ErrFileNotExist {
 		t.logger.Panic(err)
 		return t.key
 	}
 
-	keyValue := getValue(t.loaded[locale][group], keyPart)
-	// if the key is not found in the current locale group, we will try to load the
-	// fallback locale and try to retrieve the translation for the given key.
+	keyValue := getValue(t.loaded[locale]["*"], key)
+	// if the key is not found `{locale}.json` file, we will try to find it in
+	// the group files.
 	if keyValue == nil {
```

Comment:
We can change this line to `if keyValue != nil {`, change the location of L120~L132 and L98~L117, the logic will be clearer.